### PR TITLE
Merge Livesite and Fix Navbar

### PR DESCRIPTION
**Pull request by rsquared226**
@helloimali did all of the work but I messed up the git history lmao.

I tested the changes and the only thing that wasn't working was the navbar on everything but the livesite. I spent a ton of time trying to figure out where the specific problem was but I could only fix it by reverting `app.component.html`.

I tried to keep his style for the most part by not altering some of the CSS tho.

### CODE REVIEW

File: user-registration-app/src/app/app.component.css [hunk 1]
```diff
@@ -58,7 +58,9 @@ p {
 }
 
 nav {
-  background-color: #B1C8CB;
+  padding-top: 50px;
+  background-color: rgba(0,0,0,0.0);
+  box-shadow: 0 0px 0px 0 rgba(0, 0, 0, 0.0), 0 0px 0px 0 rgba(0, 0, 0, 0.0), 0 0px 0px 0px rgba(0, 0, 0, 0);
```
Comment (PScottZero) [inline user-registration-app/src/app/app.component.css:63]: replace background color and box-shadow with:
```
background: transparent;
box-shadow: none;
```

File: user-registration-app/src/app/views/live-workshop/live-workshop.component.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+.card-image {
+  max-height: 100% !important;
+  overflow: visible;
+  align: center;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+}
+
+.card-image img {
```
Comment (PScottZero) [inline user-registration-app/src/app/views/live-workshop/live-workshop.component.css:22]: The images mostly appear to either be squished or stretched (they don't maintain their aspect ratio), I would look into this article for a fix: https://medium.com/@biancapower/how-to-make-a-rectangle-image-a-circle-in-css-2f392bc9abd3

**Review by PScottZero — COMMENTED** [review]
The navbar seems to be colliding with various elements of the live website. Also, the login/logout button can go off screen.

image 1
image 2
image 3

**Comment by helloimali** [comment]
This is zoomed in at 125%. Anything above will go into the mobile view with the 3 bars/hamburger menu (since it assumes it's mobile. 
Screen Shot 2020-10-14 at 1 32 12 AM
Screen Shot 2020-10-14 at 1 33 01 AM

Below is normal viewing
Screen Shot 2020-10-14 at 1 34 07 AM
Screen Shot 2020-10-14 at 1 34 15 AM

File: user-registration-app/src/app/views/live-workshop/live-workshop.component.css (hunk 2 above)
Comment (helloimali) [reply user-registration-app/src/app/views/live-workshop/live-workshop.component.css:22]: oh yea, I was gonna fix that up over the summer and then totally forgot - thanks for the heads up!!

**Comment by PScottZero** [comment]
Only nitpick is I think the MLH badge should be nudged just a bit to the left. Its a bit too close to the screen border.

**Pull request merged**